anjani/filters.py

```diff
@@ -239,8 +239,13 @@ async def func(flt: CustomFilter, client: Client, message: Message) -> bool:
         if priv:
             return False
 
-        if message.sender_chat:  # Anonymous Admin
-            return True
+        if message.sender_chat:
+            if message.sender_chat.id == message.chat.id: # Anonymous Admin
+                return True
+
+            curr_chat = await client.get_chat(message.chat.id)
+            if curr_chat.linked_chat and message.sender_chat.id == curr_chat.linked_chat.id:  # Linked Channel Owner
```

Comment:
Lgtm. Can you also add a forward check in case if the linked channel (*channel admin) broadcast a command and the command forwarded to the group that makes it trigger this filter.

Reply:
also `return False` after this if block, so it will not overflow to the code bellows.